homeassistant/components/backup/manager.py

```diff
@@ -356,9 +356,7 @@ async def async_get_backup(
     async def async_remove_backup(self, backup_id: str, **kwargs: Any) -> None:
         """Remove a backup."""
         for agent in self.backup_agents.values():
-            if not hasattr(agent, "async_remove_backup"):
-                continue
-            await agent.async_remove_backup(backup_id)
+            await agent.async_delete_backup(backup_id)
```

Comment:
Should these be run in parallel or sequence?

Reply:
Let's do that in a follow up.